.github/workflows/publish.yaml

```diff
@@ -0,0 +1,64 @@
+name: Publish
+
+on:
+  push:
+    tags:
+    - '*'
+  workflow_dispatch:
+    inputs:
+      docker_tag_prefix:
+        description: "Enter the Docker Tag Prefix (e.g adhoc-test)"
+        required: true
+        type: string
+env:
+  ECR_REPOSITORY_URL: 673156464838.dkr.ecr.us-west-2.amazonaws.com
+  ECR_REPOSITORY_NAME: block-producers-uptime
```

Comment:
Let's change this to `uptime-service-backend` because `block-producers-uptime` is deprecated

Reply:
This will also require to either set `IMAGE_NAME` or change the default value of the variable in `scripts/build.sh`

Reply:
@Smorci I have updated the docker image name to `uptime-service-backend`. 
However, there are still multiple occurence of `block-producer-uptime` in the code.
```
src/itn_uptime_analyzer/README.md
134:    "s3_bucket": "673156464838-block-producers-uptime",

src/itn_uptime_analyzer/constants.go
8:    return config.Aws.AccountId + "-block-producers-uptime"
```
Ideally, it should have been treated in a separate ticket that covers all the renaming